server/vcap-services.es6

```diff
@@ -52,3 +52,13 @@ vcapConstants.eAuthCert = eAuthService.credentials.cert;
 vcapConstants.eAuthPrivateKey = eAuthService.credentials.privatekey;
 
 module.exports = vcapConstants;
+
+// SMTP
+let smtp = VCAPServices['user-provided'].find(element => {
+  return element.name === 'smtp-service';
+});
+vcapConstants.smtpHost = smtp.credentials.smtpserver;
+vcapConstants.smtpUsername = smtp.credentials.username;
+if (smtp.credentials.password) {
+  vcapConstants.smtpPassword = smtp.credentials.password;
```

Comment:
if there's no password defined, smtpPassword will be undefined with or without the if statement, right?

Reply:
Correct and updated